### ISSUE TITLE
Rename namespace from 'single organism phenotype' to 'single species phenotype'

### ISSUE THREAD

**Issue by CuzickA** [pombase/canto#2146]
It would be clearer to have the PHIPO branch for single species named as 'single species phenotype'.

Under this branch there are population phenotypes so 'single organism phenotype' is inaccurate.
tagging @jseager7 as discussed

**Comment by kimrutherford** [comment]
This shouldn't be a problem.  We'll need to coordinate the change so that the existing annotation and the Canto configuration is changed at the same time that the new OBO file is loaded into Canto.

**Comment by jseager7** [comment]
@kimrutherford I remember that last time we had to change PHIPO namespaces (merging the 'host phenotype' and 'pathogen phenotype' namespaces into one namespace), Canto didn't remove the old namespace, and we had to load an empty ontology containing the old namespace to force Canto to clear the terms from the database. Is this still going to be the case?

**Comment by kimrutherford** [comment]
> Is this still going to be the case?

Yep, I think so.  Maybe I could could try the ontology change and database locally to make sure everything works before you modify the main PHI-Canto.

**Comment by kimrutherford** [comment]
Hi James.

I tried this and it all seemed quite smooth.  I reloaded the OBO files and included the dummy ontology below so that single_organism_phenotype terms were correctly removed.

Then I changed
```
namespace: single_organism_phenotype
```
 to
```
namespace: single_species_phenotype
```
in three places in the annotation_type_list config.

That was all that was needed because in the database we only store the annotation type name (like "host_phenotype") in the annotations, not the ontology name and the term IDs haven't changed.

---------------------------------------------------------------------

format-version: 1.2
data-version: phipo/releases/2019-12-09/phipo-simple.owl
ontology: phipo
property_value: created:by "alaynecuzick" xsd:string
property_value: creation:date 2018-07-09T13:43:00Z xsd:string
property_value: http://purl.org/dc/elements/1.1/description "Ontology of species-neutral phenotypes observed in pathogen-host interactions." xsd:string
property_value: http://purl.org/dc/elements/1.1/title "Pathogen Host Interactions Phenotype Ontology" xsd:string
property_value: http://purl.org/dc/terms/license https://creativecommons.org/licenses/by/3.0/ xsd:string

[Term]
id: PHIPO:99999999
name: dummy
namespace: single_organism_phenotype
def: "dummy." []
